lib/sharing/gists.dart

```diff
@@ -142,8 +142,12 @@ ${styleRef}${dartRef}  </head>
       // Stamp the file content with the current description.
       List<String> lines = readmeFile.content.split('\n').toList();
       if (lines.isNotEmpty && lines.first.startsWith('# ')) {
-        lines[0] = '# ${gist.description}';
+        lines[0] = '# ${gist.description}'; 
+        //TODO: Change description to "title"
         readmeFile.content = lines.join('\n');
+        readmeFile.content = readmeFile.content.split('<pre>')[0];
```

Comment:
If there's not a `<pre>` this could get hairy. Let's just create new readme contents with the gist description (title) and summary (if there is one).